test/NuGet.Core.Tests/NuGet.PackageManagement.Test/BuildIntegration/BuildIntegratedRestoreUtilityTests.cs

```diff
@@ -7,10 +7,12 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using NuGet.Common;
```

Comment:
I'll remove that, it must have been left from when I added the test there first.